calibrate.py

```diff
@@ -70,19 +70,32 @@ def parse_args():
     parser.add_argument("-co", "--config_overwrite", default=None,
                         type=str, required=False,
                         help="JSON-formatted pipeline config object. This will be override defaults used in this script.")
-    parser.add_argument("-fv", "--field-of-view", default=71.86, type=float,
+    parser.add_argument("-brd", "--board", default=None, type=str,
+                        help="BW1097, BW1098OBC - Board type from resources/boards/ (not case-sensitive). "
+                            "Or path to a custom .json board config. Mutually exclusive with [-fv -b -w]")
+    parser.add_argument("-fv", "--field-of-view", default=None, type=float,
                         help="Horizontal field of view (HFOV) for the stereo cameras in [deg]. Default: 71.86deg.")
-    parser.add_argument("-b", "--baseline", default=9.0, type=float,
+    parser.add_argument("-b", "--baseline", default=None, type=float,
```

Comment:
This seems to not always be the case. Yes, removing it works for those `float`s here, but few lines below I have:
`parser.add_argument("-w", "--no-swap-lr", default=None, action="store_false", ...`
If I remove `default=None` there, the default changes to `True`, because of the `action` option, as described here: https://docs.python.org/3/library/argparse.html#action
And it breaks the logic below, where there are actual checks for `None`, for the incompatible arguments.
So I'd prefer to keep `default=None` for clarity, even more because of those explicit checks.
Thoughts?

Reply:
Agreed, thanks!